### PR TITLE
Sent state added

### PR DESCRIPTION
**Pull request by borjasalguero**

### CODE REVIEW

File: remote_app/js/app.js [hunk 1]
```diff
@@ -29,9 +29,20 @@ window.onload = function() {
           action:  'open'
         }, function(e, result) {
           if (e) {
-            alert('ERROR WHILE CREATING MEDIA ' + JSON.stringify(e));
+            messagesDongle.textContent = 'Error :(';
+            messagesDongle.setAttribute('color', 'red');
+            console.log('ERROR WHILE CREATING MEDIA ' + JSON.stringify(e));
+            setTimeout(function(){
+              window.close();
+            }, 500)
```
Comment (pacorampas) [inline remote_app/js/app.js]: The messgae is very fast. I think 1000 or 1500 milliseconds is better.

File: remote_app/js/app.js [hunk 2]
```diff
@@ -29,9 +29,20 @@ window.onload = function() {
...
             return;
           }
+
+          setTimeout(function(){
+            messagesDongle.textContent = 'Content sent!';
+            messagesDongle.setAttribute('color', 'dark-green');
+          }, 200);
```
Comment (pacorampas) [inline remote_app/js/app.js]: Is possible putting more time?

File: remote_app/style/app.css [hunk 3]
```diff
@@ -56,6 +56,11 @@ p#messages-dongle[color="green"] {
   color: #fff;
 }
 
+p#messages-dongle[color="dark-green"] {
+  background-color: #4B780A;
```
Comment (pacorampas) [inline remote_app/style/app.css:60]: Could you test with this green: 00FF54?

File: remote_app/js/app.js (hunk 1 above)
Comment (borjasalguero) [inline remote_app/js/app.js]: Updated to 1000 ms!

File: remote_app/js/app.js (hunk 2 above)
Comment (borjasalguero) [inline remote_app/js/app.js]: Updated to 500 ms :)

File: remote_app/style/app.css (hunk 3 above)
Comment (borjasalguero) [inline remote_app/style/app.css:60]: I will keep the 'dark-green' style, we will revisit this in the future.

**Pull request merged**